docs/guides/bundle-guide.md

```diff
@@ -124,5 +124,5 @@ The output of `duffle install` comes from the run script. `hello world` is print
 
 ## Notes and Next steps
 
-- There are alternatives to defining a custom `run` tool. See examples of more complex and different bundles [here](https://github.com/deislabs/bundles).
-- Read more about the CNAB spec in the [docs](https://github.com/deislabs/cnab-spec/blob/master/100-CNAB.md)
+- There are alternatives to defining a custom `run` tool. See examples of more complex and different bundles [here](../../examples).
```

Comment:
Just a tip, you can use absolute paths like `/examples`, which can be easier to deal with sometimes than relative references when the nesting is deep. GitHub knows that it is relative to the repo and will change it to `https://github.com/deislabs/duffle/examples`.